### PR TITLE
Add release process to CONTRIBUTION.md

### PR DESCRIPTION
**Pull request by teodor2312**
image

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,6 +50,18 @@ Our CI is not (yet) public, so it may be difficult to understand why your pull r
...
 
+## Releasing a new version
+
+The integration has workflows setup to automate the release process, by creating commits, PRs, tags and releases.
+
+The PRs created as part of the release process will need to be merged manually, but each will contain instructions inside them for what needs to be done.
+
+Whenever a new version of [datadog-ci](https://github.com/DataDog/datadog-ci) is released, this will automatically trigger a new PR to open for this repo. The PR will be named `[dep] Bump datadog-ci to {version}` and will contain the changes to update to the latest version of datadog-ci and the steps you need to follow to continue the release process.
+
+After completing the steps from the **[dep]** PR, you will have a new **[release]** PR created. When this happens, go to the PR and follow the instructions there on how to finalize the release process.
```
Comment (jakepruitt) [inline CONTRIBUTING.md:61]:
```suggestion
After completing the steps from the **[dep]** PR, a new **[release]** PR will automatically be created. When this happens, go to the PR and follow the instructions there on how to finalize the release process.
```

**Review by jakepruitt — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -50,6 +50,18 @@ Our CI is not (yet) public, so it may be difficult to understand why your pull r
 
 The code under this repository follows a format enforced by prettier, and a style guide enforced by eslint.
 
+## Releasing a new version
+
+The integration has workflows setup to automate the release process, by creating commits, PRs, tags and releases.
```
Comment (Drarig29) [inline CONTRIBUTING.md:55]:
```suggestion
The integration has workflows set up to automate the release process, by creating commits, PRs, tags and releases.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -50,6 +50,18 @@ Our CI is not (yet) public, so it may be difficult to understand why your pull r
...
+
+The PRs created as part of the release process will need to be merged manually, but each will contain instructions inside them for what needs to be done.
```
Comment (Drarig29) [inline CONTRIBUTING.md:57]:
```suggestion
The PRs created as part of the release process will need to be merged manually, and each will contain instructions inside them for what needs to be done.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -50,6 +50,18 @@ Our CI is not (yet) public, so it may be difficult to understand why your pull r
...
+
+Whenever a new version of [datadog-ci](https://github.com/DataDog/datadog-ci) is released, this will automatically trigger a new PR to open for this repo. The PR will be named `[dep] Bump datadog-ci to {version}` and will contain the changes to update to the latest version of datadog-ci and the steps you need to follow to continue the release process.
```
Comment (Drarig29) [inline CONTRIBUTING.md:59]:
```suggestion
Whenever a new version of [datadog-ci](https://github.com/DataDog/datadog-ci) is released, a new PR will automatically be created on the current repository. The PR will be named `[dep] Bump datadog-ci to {version}` and will contain the changes to update to the latest version of datadog-ci and the steps you need to follow to continue the release process.
```

**Review by Drarig29 — APPROVED** [review]
LGTM (a few nits)

**Pull request merged**